scripts/travis-doc-check.sh

```diff
@@ -0,0 +1,32 @@
+#! /bin/bash
+
+services=( auth checkin decision event mail notifications registration rsvp stat upload user )
+
+if [ "$TRAVIS_PULL_REQUEST" != "false" ] ; then
+    # get all changed files in PR
+    changed_files=`git diff --name-only $TRAVIS_BRANCH...HEAD`
+
+    # figure out which services might be a problem
+    n_changed_services=0
+    changed_services_list=""
+    for service in ${services[*]} ; do
+	service_upper=${service^}
+	
+	# check if any file in the service has been changed and if the doc page has not been changed
+	if [[ $changed_files =~ "services/${service}" ]] \
```

Comment:
Can we simplify this logic into separate variables which make it clear what each expression signifies?
This is currently hard to read.